README.md

```diff
@@ -86,6 +86,11 @@ Currently yay Depends on:
 
 * Yay does not display colored output. How do I fix it?
   Make sure you have the `Color` option in your `/etc/pacman.conf` [#123](https://github.com/Jguer/yay/issues/123)
+  
+* Yay is not asking to select packages to install (issue [#554](https://github.com/Jguer/yay/issues/554))
+
+  The default behavior was changed from [v8.918](https://github.com/Jguer/yay/releases/tag/v8.918) (see: [3bdb534](https://github.com/Jguer/yay/commit/3bdb5343218d99d40f8a449b887348611f6bdbfc)).
+  To restore such behavior use the `--combinedupgrade` option, example `yay -Suy --combinedupgrade`, note that in this case the usege of the native `pacman` is partial.
```

Comment:
I'd probably prefer `. Note: this causes (link to wiki)[native pacman] to become partial`

You can also rebase all the commits together when you're done.